pybrake/test_celery_integration.py

```diff
@@ -0,0 +1,47 @@
+import json
+import threading
+from time import sleep
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from .test_celery import raiseException
+
+class Handler(BaseHTTPRequestHandler):
+    def do_POST(self):
+        content_length = int(self.headers['Content-Length'])
+        post_data = self.rfile.read(content_length).decode('utf-8')
+
+        global notice
+        notice = json.loads(post_data)
+
+        self.send_response(200)
+        self.wfile.write('{"id":"1"}'.encode('utf-8'))
+
+def test_celery_integration():
+    server_address = ('', 8080)
+    server = HTTPServer(server_address, Handler)
+
+    httpd_thread = threading.Thread(target=server.serve_forever)
+    httpd_thread.setDaemon(True)
+    httpd_thread.start()
+
+    try:
+        result = raiseException.delay()
```

Comment:
use raiseException.apply to execute task locally - http://docs.celeryproject.org/en/latest/reference/celery.app.task.html